build/_build.csproj

```diff
@@ -2,15 +2,16 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <NoWarn>CS0649;CS0169</NoWarn>
     <NukeRootDirectory>..</NukeRootDirectory>
     <NukeScriptDirectory>..</NukeScriptDirectory>
+    <NukeTelemetryVersion>1</NukeTelemetryVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nuke.Common" Version="5.3.0" />
-    <PackageReference Include="Nuke.OctoVersion" Version="0.2.453" />
+    <PackageReference Include="Nuke.Common" Version="6.1.2" />
+    <PackageDownload Include="OctoVersion.Tool" Version="[0.2.1058]" />
```

Comment:
Why does updating to .NET 6 mean we have to change this package? Are they actually different packages, or has the same package been renamed?

Reply:
I did some recon on this. I'm mostly just saying this phonetically, but [the `Nuke.OctoVersion` functionality has been added to Nuke](https://github.com/OctopusDeploy/OctoVersion/pull/38), and it just needs the tool download now. Could probably get away without upgrading this, but worthwhile just getting that's related up to date.